### PR TITLE
Edit sticky app bar & header

### PR DESCRIPTION
**Pull request by 0shar0**

### CODE REVIEW

File: src/models/headerVisibility/selecor.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { RootStateType } from '../rootReducer';
+import { IVisibilityState } from './types';
+
+export const selectHeaderVisibility = (
```
Comment (michalenr) [inline src/models/headerVisibility/selecor.ts:4]: typo in the filename

File: src/models/headerVisibility/reducer.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+};
+
+export const visibilitySlice = createSlice({
+  name: 'HeaderVisibility',
+  initialState,
+  reducers: {
+    makeHeaderVisible: (state) => {
+      state.visibility = true;
+    },
+    makeHeaderUnVisible: (state) => {
```
Comment (michalenr) [inline src/models/headerVisibility/reducer.ts:16]: invisible

File: src/models/headerVisibility/selecor.ts (hunk 1 above)
Comment (0shar0) [reply src/models/headerVisibility/selecor.ts]: fixed

File: src/models/headerVisibility/reducer.ts (hunk 2 above)
Comment (0shar0) [reply src/models/headerVisibility/reducer.ts]: fixed

**Review by michalenr — APPROVED** [review]

**Review by niksonax — APPROVED** [review]

**Pull request merged**